### PR TITLE
[DOCS][WIP] Creating a local cluster

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
A draft right now to get some discussion and questions answered. Can fix merge issues later.

### CODE REVIEW

File: docs/_doc/Creating-a-local-cluster.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+---
+title: Creating a local cluster
+---
+
+<!-- TODO: Redirects -->
```
Comment (ChrisChinchilla) [inline docs/_doc/Creating-a-local-cluster.md]: With the path changes, do we need to setup redirects

File: docs/_doc/Creating-a-local-cluster.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+> In the first link, the first section "Setting up multiple nodes" does not necessarily imply that those nodes will be connected in any way. They could literally be separate nodes that have no knowledge of each other. Perhaps serving as an array of nodes for some local testers, for example.
```
Comment (ChrisChinchilla) [inline docs/_doc/Creating-a-local-cluster.md]: I am confused by these comments, doesn't `admin.addPeer(enodeUrlOfFirstInstance)` mean they are aware of each other? Or did I end up removing the content that makes them unaware?

File: docs/_doc/Creating-a-local-cluster.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+To run multiple ethereum nodes locally, you have to make sure:
+
+-   each instance has a separate data directory (`--datadir`)
+-   each instance runs on a different port (both ETH and RPC) (`--port and --rpcport`)
+-   in case of a cluster the instances must know about each other
+-   the IPC endpoint is unique or the IPC interface is disabled (`--ipcpath or --ipcdisable`)
+
+Create the temporary directories needed by geth to store data and save logs:
+
+<!-- TODO: Why `60` -->
```
Comment (ChrisChinchilla) [inline docs/_doc/Creating-a-local-cluster.md]: Why 60? Completely arbitrary?

File: docs/_doc/Creating-a-local-cluster.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+}]
+```
+
+### Scripting cluster creation
+
+You can automate the creation a local cluster of nodes, including account creation which is needed for mining. See the [`gethcluster.sh`](https://github.com/ethersphere/eth-utils/blob/master/gethcluster.sh) script, and the [README](https://github.com/ethersphere/eth-utils/blob/master/README.md) for usage and examples.
+
+### Monitoring your nodes
+
+<!-- TODO: Bring in? -->
```
Comment (ChrisChinchilla) [inline docs/_doc/Creating-a-local-cluster.md]: The `Setting-up-monitoring-on-local-cluster` page isn't that long, I think we should just bring it into this guide, thoughts?

File: docs/_doc/Private-network.md [hunk 5]
```diff
@@ -1,26 +1,74 @@
 ---
 title: Private network
 ---
-An Ethereum network is a private network if the nodes are not connected to the main
-network nodes. In this context private only means reserved or isolated, rather than
+
```
Comment (ChrisChinchilla) [inline docs/_doc/Private-network.md]: I moved a section from the page above into this page as they both dealt with the whole bootnode process. I wonder if this was more the "advanced" steps that @fjl mentioned, so left alone for now. Maybe it isn't, but it's two different approaches.

I have done minimal editing to this page so far.

File: docs/_doc/Creating-a-local-cluster.md (hunk 3 above)
Comment (holiman) [reply docs/_doc/Creating-a-local-cluster.md]: yep, looks very arbitrary to me

File: docs/_doc/Creating-a-local-cluster.md (hunk 2 above)
Comment (holiman) [reply docs/_doc/Creating-a-local-cluster.md]: Yeah, that should indeed make them aware of eachother, imo

**Comment by fjl** [comment]
Honestly, I think this whole page is a mess and doesn't really need to exist anymore.
The 'local cluster' page was created before puppeth came into existence. It is now much easier
to set up multiple nodes using puppeth. If someone wants to set up a network 'by hand', or using ansible, the new private network documentation covers that.

**Comment by ChrisChinchilla** [comment]
@fjl A "mess" is a bit strong. I did speak with one of the team about the existing content and puppeth was never mentioned, also had no idea you were working on the same content.

Anyway, probably the best approach now is that I take some of the approach to what I wrote here, ad review your recent changes with a beginners eye in mind, and close this PR.

**Comment by ChrisChinchilla** [comment]
Closing for now, in favour of using pupeth for a getting started guide.

**Pull request closed without merging**